app/assets/javascripts/hw_combobox/models/combobox/dialog.js

```diff
@@ -11,6 +11,10 @@ Combobox.Dialog = Base => class extends Base {
     if (window.visualViewport) {
       window.visualViewport.addEventListener("resize", this._resizeDialog)
     }
+
+    if (this._isiOS) {
+      this.dialogTarget.style.position = "absolute"
```

Comment:
I'd be surprised if this works for every case. But I can't seem to find a way to make `position: fixed` work everywhere.

Reply:
On second thought, we should find a fix that works everywhere. I'll keep looking.